langchain/src/agents/tool_calling/index.ts

```diff
@@ -40,6 +40,10 @@ export type CreateToolCallingAgentParams = {
    * allowing streaming of intermediate steps. Defaults to true.
    */
   streamRunnable?: boolean;
+  /**
+   * Optional kwargs to pass to the underlying model.
+   */
+  kwargs?: Record<string, unknown>;
```

Comment:
Lets use the base interface, combined w/ `Record<string, any>` (not 100% sure if you need the `Partial<>`)
```suggestion
  kwargs?: BaseChatModelCallOptions & Partial<Record<string, any>>;
```